app/screens/earns-map-screen/earns-map-screen.tsx

```diff
@@ -34,11 +32,10 @@ import RightTodo from "./right-section-to-do-01.svg"
 import TextBlock from "./text-block-medium.svg"
 import { useQuizServer } from "./use-quiz-server"
 import { makeStyles, useTheme } from "@rneui/themed"
-
-const BottomOngoingEN = React.lazy(() => import("./bottom-ongoing-01.en.svg"))
-const BottomOngoingES = React.lazy(() => import("./bottom-ongoing-01.es.svg"))
-const BottomStartEN = React.lazy(() => import("./bottom-start-01.en.svg"))
-const BottomStartES = React.lazy(() => import("./bottom-start-01.es.svg"))
+import BottomOngoing from "@app/components/translateable-svgs/bottom-ongoing"
+import BottomStart from "@app/components/translateable-svgs/bottom-start"
+import BottomOngoingES from "./bottom-ongoing-01.es.svg"
+import BottomStartES from "./bottom-start-01.es.svg"
```

Comment:
do we still need those 2?

Reply:
Nope. They're there while I was waiting on as response for the alternative solution. I'll remove them since it seems like you've decided